### ISSUE TITLE
Totuusarvokenttä tai valintakenttä tulostyypiksi

### ISSUE THREAD

**Issue by ZeiP** [partio-scout/kipa#7]
Tulostyypiksi voisi lisätä totuusarvokentän ja/tai valintakentän, jolla eri arvot antavat eri numerot (suuri rike = -1, pieni rike = -0,5, ei rikettä = 0 tai ruksi = 1, ei ruksia = 0). Ehkäisee syöttövirheitä, koska nykyisellään vastaavassa pitää toivoa että kukaan ei tunge totuusarvokenttään ykkösen sijaan kakkosta.

**Comment by Pena86** [comment]
Totta, nuo tulostyypit olisivat hyvä lisä ohjelmaan, vaikkakin minua epäilyttää valintakentän käytettävyys mobiililaitteilla.
Nykyisellään vastaavan toiminnan voi toteuttaa laittamalla laskentakaavaksi joko:
min(abs(a),1)
tai:
if(a,1,0)
Näin tulos a saa ensimmäisessä tapauksessa arvoja 0.0 ... 1.0 ja jälkimmäisessä joko 0 tai 1 (mutta tarkistustuloksissa pitää todennäköisesti olla sama syöte molemmissa. En tätä testannut).
Tätä kun testasin, niin näyttäisi siltä, että tuloksiin ei voi syöttää negatiivista arvoa, eli tuo abs -funktio on turha tuolla välissä... Tulipa vielä jotain uutta minullekkin.
